api4/job.go

```diff
@@ -167,7 +167,12 @@ func getJobs(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	w.Write([]byte(model.JobsToJson(jobs)))
+	js, jsonErr := json.Marshal(jobs)
```

Comment:
What does `js` stand for? It's not obvious to me.

Reply:
It stands for json, given the full name is taken by the package.

Reply:
0/5 maybe use `out` or `jsonOut` as it's more obvious.

Reply:
I am going to defer this as it's a minor detail and proper replacement could take a bit.